admin-auth/src/main/kotlin/com/epam/drill/admin/auth/route/UserAuthenticationRoutes.kt

```diff
@@ -16,60 +16,78 @@
 package com.epam.drill.admin.auth.route
 
 import com.epam.drill.admin.auth.exception.UserNotAuthenticatedException
+import com.epam.drill.admin.auth.service.TokenService
 import com.epam.drill.admin.auth.service.UserAuthenticationService
-import com.epam.drill.admin.auth.view.ApiResponse
-import com.epam.drill.admin.auth.view.ChangePasswordForm
-import com.epam.drill.admin.auth.view.LoginForm
-import com.epam.drill.admin.auth.view.RegistrationForm
+import com.epam.drill.admin.auth.view.*
 import io.ktor.application.*
 import io.ktor.auth.*
 import io.ktor.http.*
 import io.ktor.request.*
 import io.ktor.response.*
 import io.ktor.routing.*
+import kotlinx.serialization.Serializable
 import org.kodein.di.DI
 import org.kodein.di.DIAware
 import org.kodein.di.instance
 
 class UserAuthenticationRoutes(override val di: DI) : DIAware {
     private val app by instance<Application>()
-    private val service by instance<UserAuthenticationService>()
+    private val authService by instance<UserAuthenticationService>()
+    private val tokenService by instance<TokenService>()
 
     init {
         app.routing {
+            loginRoute()
             signInRoute()
             signUpRoute()
             updatePasswordRoute()
         }
     }
 
-    fun Route.signInRoute() {
+    private fun Route.signInRoute() {
         post("sign-in") {
             val form = call.receive<LoginForm>()
-            val token = service.signIn(form)
-            call.response.header(HttpHeaders.Authorization, token.token)
-            call.respond(HttpStatusCode.OK, token)
+            val view = authService.signIn(form)
```

Comment:
Pick meaningful val name
Using `view` name here is so generic - it obscures intent
It's like saying "this is return value", or "this is string"